lib/buttons-and-links/buttons.scss

```diff
@@ -41,7 +41,7 @@ a.button {
 }
 
 .link,
-.link .label > span > span {
```

Comment:
We may need to add the `link-text` class to other components which use the `buttonClasses` mixin property like

https://github.com/learningequality/kolibri-design-system/blob/64f92dbeb39d92035ed4f4112928f35b95863bab/lib/buttons-and-links/KExternalLink.vue#L20

and 

https://github.com/learningequality/kolibri-design-system/blob/64f92dbeb39d92035ed4f4112928f35b95863bab/lib/buttons-and-links/KButton.vue#L25